src/dataSources/cloudFirestore/milestone.js

```diff
@@ -0,0 +1,70 @@
+import debug from 'debug';
```

Comment:
dumb question but since we added the milestones properly now, did we also fix the date issue we had with them?

Reply:
@csell5 What is the date issue you mention?